readme.md

```diff
@@ -81,27 +81,54 @@ to [run the E2E test](./LOCAL_TESTING.md).
 [Discord](https://discord.com/invite/zetachain) |
 [Telegram](https://t.me/zetachainofficial) | [Website](https://zetachain.com)
 
+## Releases
 
-## Creating a Release Candidate
-Creating a release candidate for testing is a straightforward process. Here are the steps to follow:
+To start a new major release, begin by creating and pushing a `release/` branch based on `develop`.
 
-### Steps
- - Step 1. Create the release candidate tag with the following format (e.g., vx.x.x-rc) ex. v11.0.0-rc.
- - Step 2. Once a RC branch is created the automation will kickoff to build and upload the release and its binaries.
+<details>
+<summary>Example Commands</summary>
+
+```bash
+git fetch
+git checkout -b release/v15 origin/develop
+git push origin release/v15
+```
+
+</details>
+
+Most changes should first be merged into `develop` then backported to the release branch via a PR.
+
+<details>
+<summary>Example Commands to Backport a Change</summary>
+
+```bash
+git fetch
+git checkout -b my-backport-branch origin/release/v15
+git cherry-pick <commit SHA from develop>
+git push origin my-backport-branch
+```
+
+</details>
+
+### Creating a Release Candidate
+Once a release branch has been created, you can use github actions to create a release candidate:
+1) Create the release candidate tag with the following format (e.g., vx.x.x-rc) ex. v11.0.0-rc.
+2) Push the tag and the automation will take care of the rest
```

Comment:
What do you think of creating multiple release candidates, such as rc1 and rc2, leading up to a release? 

From the workflow perspective, it would mean that, instead of backporting multiple changes onto RC1, we create RC2 instead.

We only create the `release/vXXX` branch once we are reasonably confident about the changes to minimize backporting changes into it. 

We can probably run the final RC on devnet or mock-mainnet for a few days, if needed, before cutting the release.

Overall, I want to avoid cherry-picking changes as much as possible and only do it if absolutely needed.

Reply:
@kingpinXD Creating RC tags directly off develop is fine so long as there is not content on develop that will not make it into the release branch. I will update the doc.

If we need to do RC tags for minor releases (`v16.0.1-rc1`), those would need to based on the release branch.